srcpkgs/ccache/template

```diff
@@ -17,7 +17,7 @@ checksum=66fc121a2a33968f9ec428e02f48ff4b8896fbabb759e9c09352267014dcbe65
 
 post_install() {
 	vmkdir usr/lib/ccache/bin
-	for f in gcc cc c++ g++; do
+	for f in gcc cc c++ g++ clang clang++; do
```

Comment:
`clang`/`clang++` don't have the gnu prefixed cross-compile binaries (e.g. `arm-linux-musleabihf-clang` ), So moving clang/clang++ into itns's own for loop without the additional symlink is probably better.

Reply:
@Johnnynator you were right, I changed it.